src/dropdown.ts

```diff
@@ -494,6 +496,8 @@ export default class Dropdown extends LitElement implements FormControl {
     ) {
       this.#inputElementRef.value.value =
         this.lastSelectedAndEnabledOption.label;
+
+      this.inputValue = this.lastSelectedAndEnabledOption.label;
```

Comment:
Nothing fixed because it's also set in `#onDefaulotSlotChange()`. Just ensures the field is always up to date.